app/src/main/java/com/buzzshelter/Controllers/CSVFile.java

```diff
@@ -0,0 +1,46 @@
+package com.buzzshelter.Controllers;
```

Comment:
Only activities are considered controllers
This class has a lot of business logic that can all be moved to the Model class or as a class a new class in the Model

Reply:
Fixed